### DISCUSSION TITLE
General technical questions

### DISCUSSION THREAD

**Discussion by Phiek1** [eclipse-emfcloud/emfcloud#190] [All other] [answered]
Hi!

I am wondering about some general questions for EMF Cloud below, any feedback really appreciated, many thanks!

- What is the support to different databases?

- What is the support for client/server. Requests done via API/Web?

- How is the system run - internal/external server?

- User interface possibilities - standalone software/web interface?

**Comment by eneufeld** [comment, marked as answer]
Hi @Phiek1 ,
Could you elaborate a bit more on your questions, what do you want to achieve?
In which component of the emflcoud project are you interested?
Maybe looking at this article will help: https://eclipsesource.com/blogs/2019/12/16/introducing-emf-cloud/
Or this talk: https://www.youtube.com/watch?v=qjris1CdJow 

Best

**Reply by Phiek1** [reply]
Thanks helpful will check these materials through! :)
What I guess I am looking for is an technical "overview" perhaps table of some sort of the technical capacity/features of EMF-cloud. 

Br

**Reply by eneufeld** [reply]
The technical overview is hard to achieve, as emfcloud is a set of different tools which allow you to develop modeling tools in the web.
So depending on what you want to achieve, you have to select the appropriate tools.
So the two links should give you a hint on what is possible and which tools are offered.
